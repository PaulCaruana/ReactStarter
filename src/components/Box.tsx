import styled, { css } from 'styled-components'

const Box = styled.div(
    p => css`
    background-color: green;
  `);
export default Box;
