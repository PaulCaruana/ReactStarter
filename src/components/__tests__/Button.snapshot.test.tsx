import React from 'react'
import { Button } from '../../components/Button'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Button href={'#'}>Default Button</Button>).toJSON()
  expect(tree).toMatchSnapshot()
})
