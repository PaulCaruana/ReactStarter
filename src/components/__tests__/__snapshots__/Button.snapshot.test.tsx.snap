// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="inline-flex rounded-md shadow"
>
  <a
    className="inline-flex items-center justify-center px-5 py-3 border border-transparent text-base leading-6 font-medium rounded-md focus:outline-none focus:shadow-outline transition duration-150 ease-in-out text-white bg-indigo-600 hover:bg-indigo-500"
    href="/#"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Default Button
  </a>
</div>
`;
